src/malay/AllMay.gf

```diff
@@ -3,4 +3,6 @@
 concrete AllMay of AllMayAbs =
   LangMay,
   ExtendMay
-  ** {} ;
```

Comment:
This is the correct place for any additional lexicon you want to use for testing! Just add `sing_V2 : V2` in the abstract syntax as well.